releases/release-7.5.3.md

```diff
@@ -0,0 +1,171 @@
+---
+title: TiDB 7.5.3 Release Notes
+summary: Learn about the compatibility changes, improvements, and bug fixes in TiDB 7.5.3.
+---
+
+# TiDB 7.5.3 Release Notes
+
+Release date: xx xx, 2024
+
+TiDB version: 7.5.3
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.5/quick-start-with-tidb) | [Production deployment](https://docs.pingcap.com/tidb/v7.5/production-deployment-using-tiup)
+
+## Compatibility changes
+
+- note [#issue](https://github.com/pingcap/${repo-name}/issues/${issue-id}) @[贡献者 GitHub ID](https://github.com/${github-id})
```

Comment:
```suggestion
- Add a new system table [`INFORMATION_SCHEMA.KEYWORDS`](/information-schema/information-schema-keywords.md) to display the information of all keywords supported by TiDB [#issue](https://github.com/pingcap/${repo-name}/issues/${issue-id}) @[dveeden](https://github.com/dveeden)
```
@dveeden Could you please provide the GitHub issue of this compatibility change?

Reply:
https://github.com/pingcap/tidb/pull/54219

Reply:
There are two other ones, but I don't think we need to mention those. All 3 are listed in the description of https://github.com/pingcap/docs/pull/18155